2-Regression/1-Tools/translations/README.az.md

```diff
@@ -0,0 +1,227 @@
+
+Bu bölmədə siz, Scikit-learn-də öyrənmə məqsədi üçün əvvəlcədən hazırlanmış şəkərli diabet xəstələri ilə bağlı olan kiçik məlumat massivi ilə işləyəcəksiniz. Maşın öyrənməsi modelləri dəyişənlərin kombinasiyası əsasında hansı xəstələrin müalicəyə daha yaxşı cavab verə biləcəyini müəyyən etməyə kömək edə bilər. Hətta ən sadə reqressiya modeli belə, vizuallaşdırıldıqda, nəzəri klinik sınaqlarınızı təşkil etməkdə sizə kömək edəcək dəyişənlər haqqında məlumat verə bilər.
+
+✅ Reqressiya metodlarının bir çox növü olsa da, hansını seçməyiniz axtardığınız cavabdan asılıdır. Əgər siz müəyyən yaşda olan bir insanın boyunun uzunluğunu təxmin etmək istəyirsinizsə, **ədədi dəyər** axtardığınız üçün, xətti reqressiyadan istifadə edəcəksiniz. Əgər siz bir mətbəxin veqan hesab edilib-edilməməsini tapmaqda maraqlısınızsa, onda siz logistik reqressiyadan istifadə edərdiniz deyə **kateqoriyalı tapşırıq** axtarırsınız. Logistik reqressiya haqqında daha sonra öyrənəcəksiniz. Bəzi sualları məlumatlardan soruşa biləcəyiniz bir az düşünün. Bu üsullardan hansı daha uyğun olardı.
+
+Reqressiya metodlarının çoxlu növləri mövcud olsa da, hansını seçməyiniz, axtardığınız cavabdan asıldır. Əgər insanın yaşına uyğun boyunun hündürlüyünü təxmin etmək istəyirsinizsə, bu məlumat ***ədədi dəyərə** malik olduğu üçün xətti reqressiyadan istifadə edə bilərsiniz. Yox əgər hansısa mətbəxin veqan hesab edilib edilməməli olduğunu tapmaqla maraqlanırsınızsa, deməli **kateqoriya mənimsədilməsini** axtarırsınız. Bunun üçün logistik reqressiyadan istifadə edə bilərsiniz. Logistik reqressiyalar haqqında sonradan daha çox öyrənəcəksiniz. Məlumatlarla bağlı soruşa biləcəyiniz suallar və bu metodlardan hansının daha uyğun ola biləcəyi ilə bağlı isə biraz düşünün.
+
+Gəlin bu tapşırıqla başlayaq.
+
+### Kitabxanaları köçürün
+
+Bu tapşırıq üçün biz bəzi kitabxanaları köçürəcəyik:
+- **matplotlib** lazımlı [qrafikləşdirmə alətidir](https://matplotlib.org/) və biz ondan xətti qrafikləri yaratmaq üçün istifadə edəcəyik.
+- **numpy**. [numpy](https://numpy.org/doc/stable/user/whatisnumpy.html) Python-da ədədi dataların emalı üçün faydalı bir kitabxanadır.
+- **sklearn**. Bu isə [Scikit-learn](https://scikit-learn.org/stable/user_guide.html) kitabxanasıdır.
+
+Tapşırıqda sizə kömək olması üçün kitabxanalardan bəzilərini köçürün.
+
+1. Aşağıdakı kodu yazaraq köçürmələri edək:
+
+   ```python
+   import matplotlib.pyplot as plt
+   import numpy as np
+   from sklearn import datasets, linear_model, model_selection
+   ```
+
+   Yuxarıda siz `matplotlib`-i, `numpy`-ı və `sklearn`-dən `datasets`, `linear_model` və `model_selection`-ı köçürürsünüz. `model_selection` datanı öyrədilmə və test massivlərinə bölmək üçün istifadə olunur.
+
+### Şəkərli diabet data massivi
+
+The built-in [diabetes dataset](https://scikit-learn.org/stable/datasets/toy_dataset.html#diabetes-dataset) includes 442 samples of data around diabetes, with 10 feature variables, some of which include:
```

Comment:
this sentence is left in English

Reply:
I have forgot to delete it. Its translation was in the next line